service/src/main/java/com/codeforcommunity/processor/ProtectedUserProcessorImpl.java

```diff
@@ -211,4 +230,27 @@ public void changePrivilegeLevel(
       throw new WrongPasswordException();
     }
   }
+
+  @Override
+  public void createChildUser(JWTData userData, NewChildRequest newChildRequest) {
+    isAdminCheck(userData.getPrivilegeLevel());
+
+    db.transaction(configuration -> {
```

Comment:
@rymaju Is this what you meant by wrapping it with a transaction?

Reply:
Sure! - ryan